foodsaving/template_previews/views.py

```diff
@@ -44,8 +44,12 @@ def shuffle_groups():
     return Group.objects.order_by('?')
 
 
-def random_message():
-    return ConversationMessage.objects.exclude_replies().order_by('?').first()
+def random_messages():
+    conversation = ConversationMessage.objects.order_by('?').first().conversation
+    messages = conversation.messages.exclude_replies().all()
+    if len(messages) > 5:
+        messages = messages[:4]
```

Comment:
Why not a limit on the query?

Reply:
It is a limit on the queryset: https://docs.djangoproject.com/en/dev/topics/db/queries/#limiting-querysets
But I need to check for the length anyways, as there might be less messages than requested. Maybe the limit is not needed at all though. But then the email preview might get a bit long

Reply:
But `len(messages)` will evaluate the query then count the entries, no?